napari/layers/shapes/_tests/test_triangulation.py

```diff
@@ -8,8 +8,8 @@
 ac = pytest.importorskip('napari.layers.shapes._accelerated_triangulate')
 
 
-@pytest.fixture
-def _disable_jit(monkeypatch):
+@pytest.fixture(params=[False, True])
+def _disable_jit(monkeypatch, request):
```

Comment:
```suggestion
def _disable_jit(request):
```
monkeypatch not used, I think? at least I don't see it + tests still pass for me